web/src/search/input/LazyMonacoQueryInput.scss

```
@@ -0,0 +1,8 @@
+@import './MonacoQueryInput';
+
+// Match Monaco styling for the intermediate input displayed while
+// lazy-loading the MonacoQueryInput, to avoid unstable UI.
+.lazy-monaco-query-input--intermediate-input {
+    height: 100%;
+    font-size: 12px;
```

(I assume this can't be deleted because maybe `form-control` overrides it)
```suggestion
    font-size: $code-font-size;
```

---

Updated here, and also in code.scss 🙂 https://github.com/sourcegraph/sourcegraph/pull/10029/commits/3b3d265fe9836008e98121af4df2a48abfcc0b9f